### ISSUE TITLE
Improve UI/UX

### ISSUE THREAD

**Issue by bhavika2502** [upes-open/OSoC-Vaccination-Recording-System#15]

**Comment by AmanKumarVlgs** [comment]
Hey @bhavika2502 can you please explain the project, I'm interested

**Comment by bhavika2502** [comment]
Sure, this is basically a vaccination recording system to keep a track of all vaccinations done in the past as well as the ones might be due in future. Reminders will be sent for due via mail. It can be used to schedule appointments etc. It will also be linked to the google calender

**Comment by AmanKumarVlgs** [comment]
Got it, So @bhavika2502 have to design Figma or code this also? If figma please assign me this task

**Comment by AmanKumarVlgs** [comment]
Hey @bhavika2502 I would love to work on this issue, I can design
- Dashboard
- Login page
Should I create a new issue or you will assign me this task?

**Comment by bhavika2502** [comment]
Hi! @AmanKumarVlgs   There are issues already created and being worked on, I can assign you those

**Comment by AmanKumarVlgs** [comment]
Yeah please assign me, can you please share issue id or link

**Comment by bhavika2502** [comment]
Okay, but this is improvement, issue to be completed once the front-end is complete

you can get assigned to other issues if you wish to work immediately @AmanKumarVlgs

**Comment by AmanKumarVlgs** [comment]
Yeah thanks for assigning me this, I am working on other issues related to this I'll do it.
